custom/icds_reports/reports/lady_supervisor.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from datetime import datetime
+
+from django.db.models.aggregates import Sum
+from django.utils.translation import ugettext as _
+
+from corehq.util.quickcache import quickcache
+from custom.icds_reports.messages import lady_supervisor_number_of_vhnds_observed_help_text, \
+    lady_supervisor_number_of_beneficiaries_visited_help_text, lady_supervisor_number_of_awcs_visited_help_text
+from custom.icds_reports.models.views import AggLsMonthy
+from custom.icds_reports.utils import get_value, apply_exclude
+
+
+@quickcache(['domain', 'config', 'show_test'], timeout=30 * 60)
+def get_lady_supervisor_data(domain, config, show_test=False):
+
+    def get_data(date, filters):
+        queryset = AggLsMonthy.objects.filter(
+            month=date, **filters
+        ).values(
+            "aggregation_level"
+        ).annotate(
+            number_of_unique_awc_vists=Sum("unique_awc_vists"),
```

Comment:
since these columns are already sums, what are we summing here?

Reply:
That's right, it is not necessary. Removed it.